AirCasting/AuthViews/KeychainStorage.swift

```diff
@@ -71,11 +76,11 @@ final class KeychainStorage {
         return string
     }
     
-    public func getUsername() -> String {
+    public func getProfileData(_ from: UserData) -> String {
```

Comment:
maybe let's use external and internal parameter as using the `from` inside code is kind of tricky